app/build.gradle

```diff
@@ -24,7 +24,7 @@ android {
         applicationId "com.teamltt.carcare"
         minSdkVersion 15
 
-        targetSdkVersion 23
+        targetSdkVersion 22
```

Comment:
Not sure we can decrease the `targetSdkVersion` here without causing problems with gradle dependencies. Below, in this file, our support dependencies are v#-23.4.0. I believe the 23 is supposed to be the same `targetSdkVersion`.